controllers/UserGroups.php

```diff
@@ -18,33 +18,8 @@ class UserGroups extends Controller
         \Backend\Behaviors\ListController::class
     ];
 
-    /**
-     * @var array `FormController` configuration.
-     */
-    public $formConfig = 'config_form.yaml';
-
-    /**
-     * @var array `ListController` configuration.
-     */
-    public $listConfig = 'config_list.yaml';
-
-    /**
-     * @var array `RelationController` configuration, by extension.
-     */
-    public $relationConfig;
```

Comment:
This could technically be a breaking change for plugins that extend this controller without using the addDynamicProperty() approach

Reply:
See https://github.com/wintercms/wn-user-plugin/commit/0852e194d9b4a88364201612462f39dd41d2bb74. @firemankurt are you still using this?

Reply:
> This could technically be a breaking change for plugins that extend this controller without using the addDynamicProperty() approach

Good point, I was actually thinking about this myself. I'll revert this part.

Reply:
I'm fine with only keeping the relationConfig property, the other two should be fine to get rid of.

Reply:
That might create problems with plugins we don't know about. Doesn't really hurt keeping those.